lib/rules/no-ref-as-operand.js

```diff
@@ -46,6 +46,9 @@ module.exports = {
         messageId: 'requireDotValue',
         data: {
           method: data.method
+        },
+        fix(fixer) {
+          return [fixer.insertTextAfter(node, '.value')]
```

Comment:
If you want to return a single fix data, you don't need to use an array.

```suggestion
          return fixer.insertTextAfter(node, '.value')
```